src/utils.js

```diff
@@ -30,6 +30,20 @@ function arrayMax( array ) {
 
 }
 
+function arrayContainsOver( array, value ) {
```

Comment:
I find this function name confusing. Maybe something more specific like `needsUint32()`? The function would return a boolean then.

Reply:
Does `indicesNeedsUint32Array( indices )` look better?

Reply:
If you want a prefix it should be:
```js
arrayNeedsUint32()
```

Reply:
Thanks. I have fixed it.